product_docs/docs/eprs/7.0/03_installation/03_installing_rpm_package.mdx

```diff
@@ -260,8 +254,43 @@ Successful installation of Replication Server results in the creation of directo
 
 If you have an existing Replication Server RPM installation, you can use yum to upgrade your repository configuration file and update to a more recent product version. To update the `edb.repo` file, assume superuser privileges and enter:
 
-> `yum upgrade edb-repo`
+`yum upgrade edb-repo`
 
 `yum` will update the `edb.repo` file to enable access to the current EDB repository, configured to connect with the credentials specified in your `edb.repo` file. Then, you can use yum to upgrade any installed packages:
 
-> `yum upgrade ppas-xdb*`
+`yum upgrade ppas-xdb*`
+
+See [Upgrading from a Replication Server RPM Package Installation](../10_appendix/02_upgrading_to_xdb6_2/03_upgrading_with_xdb_rpm_package/) for details.
+
+## Updating a Bitrock Installer Installation
+
+If you are upgrading from a 6.2.x version of Replication Server that was installed using the Replication Server Bitrock installer, perform the following steps:
```

Comment:
You can. upgrade from a 6.2.x version of Replication Server that was installed using the Replication Server Bitrock installer.